src/main/java/edu/iis/powp/command/CommandSetPosition.java

```diff
@@ -0,0 +1,19 @@
+package edu.iis.powp.command;
+
+import edu.iis.client.plottermagic.IPlotter;
+
+public class CommandSetPosition implements PlotterCommand {
+	private int x;
+	private int y;
+
+	public CommandSetPosition(int x, int y) {
+		this.x = x;
+		this.y = y;
+	}
+
+	@Override
+	public void execute(IPlotter iPlotter) {

```

Comment:
name plotter would be better. Prefix 'I' was meant to suggest an interface, but iPlotter is an object.